tools/invoke-mermaid.sh

```diff
@@ -2,11 +2,12 @@
 set -eo pipefail
 
 cfg_file=$(dirname "$0")/mermaid.json
+cfg_p_file=$(dirname "$0")/mermaid-p.json
```

Comment:
nit: Spell out what `p` is (puppeteer) in this variable name and (more importantly) the filename

(Mermaid uses puppeteer? That probably explains our determinism issues 🙁)

Reply:
Done! And indeed, and puppeteer also gets in way for some other things by running them through the CSS engine :S